CMakeLists.txt

```diff
@@ -275,12 +275,17 @@ add_custom_target(
 )
 
 
+set(VIAMCPPSDK_ABSL_VERSION_MINIMUM)  # Someday?
 set(VIAMCPPSDK_BOOST_VERSION_MINIMUM 1.71)
 set(VIAMCPPSDK_GRPC_VERSION_MINIMUM 1.30.2)
 set(VIAMCPPSDK_PROTOBUF_VERSION_MINIMUM 3.12.4)
 set(VIAMCPPSDK_XTL_VERSION_MINIMUM 0.7.2)
 set(VIAMCPPSDK_XTENSOR_VERSION_MINIMUM 0.24.3)
 
+# Time to find absl. Note that the version min value is currently empty,
+# because absl doesn't use a versioning scheme that works here.
+find_package(absl ${VIAMCPPSDK_ABSL_VERSION_MINIMUM} REQUIRED)
```

Comment:
The new abseil requirement is a little annoying. We need to explicitly use it to get hold of the Base64 manipulation routines because in very modern protobuf the Base64 routines were removed, presumably in favor of the abseil ones. Rather than trying to use the ones from protobuf for old protobuf and abseil for new protobuf was proving to be a nuisance, so I just switched everything over to use abseil.